Postgres test sometimes failed by duplicated table name

### Summary

The Postgres test has been failed with "already exists" error. Therefore, PR testing is not accurate.


For example, the test of #318 has been failed with [this error](https://github.com/fedify-dev/fedify/actions/runs/16693145488/job/47253595690).

```
 ERRORS 

./postgres/mq.test.ts (uncaught error)
error: (in promise) PostgresError: type "fedify_message_test_onje0zml" already exists
    at ErrorResponse (file:///home/runner/.cache/deno/npm/registry.npmjs.org/postgres/3.4.7/src/connection.js:794:26)
    at handle (file:///home/runner/.cache/deno/npm/registry.npmjs.org/postgres/3.4.7/src/connection.js:480:6)
    at Socket.data (file:///home/runner/.cache/deno/npm/registry.npmjs.org/postgres/3.4.7/src/connection.js:315:9)
    at Socket.emit (ext:deno_node/_events.mjs:436:20)
    at addChunk (node:_stream_readable:452:12)
    at readableAddChunkPushByteMode (node:_stream_readable:413:3)
    at Socket.Readable.push (node:_stream_readable:308:48)
This error was not caught from a test and caused the test runner to fail on the referenced module.
It most likely originated from a dangling promise, event/timeout handler or top-level code.

 FAILURES 

./postgres/mq.test.ts (uncaught error)

FAILED | 22182 passed (289 steps) | 2 failed | 2 ignored (3m4s)

error: Test failed

Stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: __libc_start_main
   9: <unknown>
Error: Process completed with exit code 1.
```

### Target Area

- `postgres/mq.test.ts`
- And other PostgreSQL tests, potentially

### Motivation

Currently, the test has sometimes been failed because of this issue. Therefore, PR testing is not working properly. After resolving this issue, PR testing will be accurate.

### Risk Assessment

_No response_

### Benefits

_No response_

--- a/packages/postgres/src/mq.test.ts
+++ b/packages/postgres/src/mq.test.ts
@@ -31,10 +31,10 @@ test("PostgresMessageQueue", { skip: dbUrl == null }, async () => {
   try {
     const messages: string[] = [];
     const controller = new AbortController();
+    await mq.initialize();
     const listening = mq.listen((message: string) => {
       messages.push(message);
     }, { signal: controller.signal });
-    await delay(500); // prevent initialization race condition
     const listening2 = mq2.listen((message: string) => {
       messages.push(message);
     }, { signal: controller.signal });

--- a/packages/postgres/src/mq.test.ts
+++ b/packages/postgres/src/mq.test.ts
@@ -34,6 +34,7 @@ test("PostgresMessageQueue", { skip: dbUrl == null }, async () => {
     const listening = mq.listen((message: string) => {
       messages.push(message);
     }, { signal: controller.signal });
+    await delay(500); // prevent initialization race condition
     const listening2 = mq2.listen((message: string) => {
       messages.push(message);
     }, { signal: controller.signal });

